### PR TITLE
Small fixes without issues

### PR DESCRIPTION
**Pull request by chlebik**
First PR for OS-project. Mostly cleanup/style, did not touch logic at all.

### CODE REVIEW

**Comment by delor** [comment]
if there are any plans to switch to gradle kotlin dsl then I would leave the `"` as it's probably first step in switching from groovy to kotlin dsl

**Comment by chlebik** [comment]
> if there are any plans to switch to gradle kotlin dsl then I would leave the `"` as it's probably first step in switching from groovy to kotlin dsl

I do not know if this is not a hidden YAGNI. For the time being I find using single quotes to indicate scalar values, and doubles to indicate placeholder substitution very clean practice, hence the changes.

File: src/main/resources/static/assets/css/toastr.min.css [hunk 1]
```diff
@@ -1 +1,6 @@
...st-bottom-full-width>div,#toast-container.toast-top-full-width>div{width:96%;margin-left:auto;margin-right:auto}.toast{background-color:#030303}.toast-success{background-color:#51A351}.toast-error{background-color:#BD362F}.toast-info{background-color:#2F96B4}.toast-warning{background-color:#F89406}.toast-progress{position:absolute;left:0;bottom:0;height:4px;background-color:#000;opacity:.4;-ms-filter:progid:DXImageTransform.Microsoft.Alpha(Opacity=40);filter:alpha(opacity=40)}@media all and (max-width:240px){#toast-container>div{padding:8px 8px 8px 50px;width:11em}#toast-container>div.rtl{padding:8px 50px 8px 8px}#toast-container .toast-close-button{right:-.2em;top:-.2em}#toast-container .rtl .toast-close-button{left:-.2em;right:.2em}}@media all and (min-width:241px) and (max-width:480px){#toast-container>div{padding:8px 8px 8px 50px;width:18em}#toast-container>div.rtl{padding:8px 50px 8px 8px}#toast-container .toast-close-button{right:-.2em;top:-.2em}#toast-container .rtl .toast-close-button{left:-.2em;right:.2em}}@media all and (min-width:481px) and (max-width:768px){#toast-container>div{padding:15px 15px 15px 50px;width:25em}#toast-container>div.rtl{padding:15px 50px 15px 15px}}
```
Comment (tdziurko) [inline src/main/resources/static/assets/css/toastr.min.css]: Do we need to modify this static css file?

File: src/test/groovy/com/jvm_bloggers/core/data_fetching/blog_posts/BlogPostFactorySpec.groovy [hunk 2]
```diff
@@ -9,6 +9,7 @@ import spock.lang.Subject
 import java.time.LocalDateTime
 import java.time.Month
 
+@Subject(BlogPostFactory)
```
Comment (tdziurko) [inline src/test/groovy/com/jvm_bloggers/core/data_fetching/blog_posts/BlogPostFactorySpec.groovy]: What is the advantage of `@Subject` at class level? And why we have to change it from annotation per field as we have it already in the code base?

File: src/test/groovy/com/jvm_bloggers/core/data_fetching/blogs/UpdateStatisticSpec.groovy [hunk 3]
```diff
@@ -5,22 +5,24 @@ import spock.lang.Subject
...
-        @Subject
         UpdateStatistic testObj = emptyStream.collect(UpdateStatistic.collector())
 
         then:
-        testObj.getTotal() == 0
-        testObj.getUpdated() == 0
-        testObj.getCreated() == 0
-        testObj.getInvalid() == 0
-        testObj.getNotChanged() == 0
+        with(testObj) {
```
Comment (tdziurko) [inline src/test/groovy/com/jvm_bloggers/core/data_fetching/blogs/UpdateStatisticSpec.groovy]: :clap:

File: src/test/groovy/com/jvm_bloggers/core/mailing/sender/EmailSendingSchedulerSpec.groovy [hunk 4]
```diff
@@ -10,14 +10,14 @@ import spock.lang.Subject
 
 import java.time.LocalDateTime
 
+@Subject(EmailSendingScheduler)
 class EmailSendingSchedulerSpec extends Specification {
 
-    LocalDateTime NOW = LocalDateTime.now();
+    private final static LocalDateTime NOW = LocalDateTime.now()
```
Comment (tdziurko) [inline src/test/groovy/com/jvm_bloggers/core/mailing/sender/EmailSendingSchedulerSpec.groovy]: Maybe only `static` would be enough?

File: src/test/groovy/com/jvm_bloggers/entities/blog_post/BlogPostRepositorySpec.groovy [hunk 5]
```diff
@@ -14,17 +14,17 @@ import java.time.LocalDateTime
 import static com.jvm_bloggers.core.rss.AggregatedRssFeedProducer.INCLUDE_ALL_AUTHORS_SET
 import static com.jvm_bloggers.entities.blog.BlogType.PERSONAL
 
+@Subject(BlogPostRepository)
 class BlogPostRepositorySpec extends SpringContextAwareSpecification {
 
+    @Autowired
+    BlogPostRepository blogPostRepository
```
Comment (tdziurko) [inline src/test/groovy/com/jvm_bloggers/entities/blog_post/BlogPostRepositorySpec.groovy]: Why this field was moved above static ones?

File: src/main/resources/static/assets/css/toastr.min.css (hunk 1 above)
Comment (chlebik) [reply src/main/resources/static/assets/css/toastr.min.css]: I did not modify it. I do not know why this is even listed as changed.

File: src/test/groovy/com/jvm_bloggers/core/data_fetching/blog_posts/BlogPostFactorySpec.groovy (hunk 2 above)
Comment (chlebik) [reply src/test/groovy/com/jvm_bloggers/core/data_fetching/blog_posts/BlogPostFactorySpec.groovy]: IMHO that is better as test/structure annotations should not interfere with test class contents. It's the same with eg. Hibernate entity mixing on the field column definition, validation conditions and for example JSON parsing exceptions.

File: src/test/groovy/com/jvm_bloggers/core/mailing/sender/EmailSendingSchedulerSpec.groovy (hunk 4 above)
Comment (chlebik) [reply src/test/groovy/com/jvm_bloggers/core/mailing/sender/EmailSendingSchedulerSpec.groovy]: As this is used only in the context of this test IMHO it is better to narrow the visibility as much as possible.

File: src/main/resources/static/assets/css/toastr.min.css (hunk 1 above)
Comment (chlebik) [reply src/main/resources/static/assets/css/toastr.min.css]: And honestly I do not know why GitHub is still complaining about it. I've reverted the file state to the last commit before all my changes and it still shows difference (and right now I've used Linux to eliminate CR/LF issues). I assume this is not that big of a deal.

**Pull request merged**